model/state.py

```diff
@@ -0,0 +1,11 @@
+import abc
```

Comment:
call the file state_space.py

not sure what `abc` is? but I assume it's like a standard module or something?